src/client.rs

```diff
@@ -120,6 +123,22 @@ impl Client {
             .await?)
     }
 
+    /// Which of the two results this returns depends on the API.
+    /// Take a look at the Kubernetes API reference.
```

Comment:
Nit: maybe add a link?

Reply:
I would but I can't find a stable link that doesn't link to a specific version.

https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/ there is no 'latest' or anything like that.